host/frontend/host-orchestrator/orchestrator/instancemanager.go

```diff
@@ -168,6 +172,20 @@ func (s *StageStartCVDServer) Run() error {
 	return err
 }
 
+type StageCreateDirIfNotExist struct {
+	Dir string
+}
+
+func (s *StageCreateDirIfNotExist) Run() error {
+	if err := os.Mkdir(s.Dir, 0755); os.IsNotExist(err) {
```

Comment:
The documentation for [`os.IsNotExist`](https://pkg.go.dev/os#IsNotExist) suggests using [`errors.Is(err, fs.ErrNotExist)`](https://pkg.go.dev/errors#example-Is).

Reply:
good observation, created b/238431258 for it as right now we are not controlling the version of `go` used when the image is built and the `fs` package was introduced in go 1.16. 

I faced the same issue before as go 1.15 is getting used when building the image because right now it's tied to the version the package manager has. I created b/236976427 to address this problem.